dmutils/apiclient.py

```diff
@@ -6,6 +6,19 @@
 logger = logging.getLogger(__name__)
 
 
+class APIError(StandardError):
```

Comment:
Should probably inherit from `HTTPError`. Also, I think `StandardError` is causing the py3 tests to fail.